### ISSUE TITLE
Documentation Uses .increments() With No Argument, but No Default Argument Value is Documented

### ISSUE THREAD

**Issue by machineghost** [knex/knex#5889]
# Missing / erroneus documentation

The entry for `.increments()` in the documentation makes no mention of a default value for its name argument, but other code in the docs uses it with no arg.  Presumably it has an undocumented default value of `id`?

# Feature discussion / request

Add one sentence to the docs for `.increments`: "Adds an auto incrementing column.  If no name is provided this will default to `"id"` ... IF that is correct.

**Comment by mercmobily** [comment]
You're correct that the documentation for `.increments()` previously didn't clearly state what happens when no argument is provided. Looking at the current documentation, this has actually been addressed.

The current documentation at https://knexjs.org/#Schema-increments now shows:

**table.increments(name, options={[primaryKey: boolean = true]})**

The documentation includes examples showing both uses:
- With a name: `table.increments('userId')`
- Without arguments: `table.increments()` (which defaults to 'id')

Looking at the actual implementation and examples throughout the docs, when no name is provided, the column name does indeed default to `'id'`. This is shown in several examples in the schema builder documentation where `table.increments()` is used without any arguments.

However, you're right that the documentation could be more explicit about this default behaviour. The method signature shows `name` as a parameter but doesn't explicitly state in the description text that it defaults to `'id'` when omitted. Adding a sentence like "If no name is provided, this defaults to 'id'" would make this clearer for developers.
